uwg_schema/refDOE.py

```diff
@@ -0,0 +1,415 @@
+from pydantic import Field, validator, root_validator, constr, conlist
```

Comment:
Yes. I used the same name we used in the UWG module, but can change it if it's confusing (i.e just reference.py).

Reply:
I know @hansukyang chose this name when he added the commercial building templates for his thesis and I don't think that he would be against a change to a more generic name to avoid confusion. Something like `bldg_template` or `ref_bldg_template` could make it a lot clearer that these templates don't have to come from the DOE.

Reply:
Sounds good, changed to ref_bldg_template